code/modules/research/designs/medical_designs.dm

```diff
@@ -830,14 +830,27 @@
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_MEDICAL
 
-/datum/design/cybernetic_ears_u
-	name = "Cybernetic Ears"
-	desc = "A pair of cybernetic ears."
-	id = "cybernetic_ears_u"
+/datum/design/cybernetic_ears_translation
+	name = "Cybernetic Translation Ears"
+	desc = "A pair of translating cybernetic ears."
+	id = "cybernetic_ears_translation"
 	build_type = PROTOLATHE | AWAY_LATHE | MECHFAB
 	construction_time = 40
-	materials = list(/datum/material/iron =SMALL_MATERIAL_AMOUNT*5, /datum/material/glass =SMALL_MATERIAL_AMOUNT*5, /datum/material/silver =SMALL_MATERIAL_AMOUNT*5)
-	build_path = /obj/item/organ/internal/ears/cybernetic/upgraded
+	materials = list(/datum/material/iron = SMALL_MATERIAL_AMOUNT*5, /datum/material/glass = SMALL_MATERIAL_AMOUNT*5, /datum/material/silver = SMALL_MATERIAL_AMOUNT*5)
```

Comment:
```suggestion
	materials = list(
		/datum/material/iron = SMALL_MATERIAL_AMOUNT*5,
		/datum/material/glass = SMALL_MATERIAL_AMOUNT*5,
		/datum/material/silver = SMALL_MATERIAL_AMOUNT*5,
	)
```

multilining is cool otherwise it'll run off the screen for some people. just looks neater too